src/django/api/templates/mail/reset_user_password_body.txt

```diff
@@ -0,0 +1,12 @@
+{% block content %}
+Hi {{ user.name }},
+
+We're sending you this email because someone requested to reset the password
+for the Open Apparel Registry account associated with this email address.
+
+You can reset your password by visiting the following link: {{ domain }}/auth/resetpassword/{{uid}}?token={{ token }}
```

Comment:
We should consider prefixing our links with `https://` so they don't default to `http`.

Reply:
Sounds good. It looks like the `protocol` option is available in these email templates, and it'll set things conditionally based on the app settings.